src/app/_services/api.service.ts

```diff
@@ -20,6 +20,24 @@ export class ApiService {
   	return rtn;
   }
 
+  // http.get needs an Authorization in header
+  setHttpHeaders(username:string, password: string) {
```

Comment:
I would just call this `getHttpHeadersWithUsernamePassword`.... or `getHttpHeadersWithBase64Authorization`.. or something like that..